<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WallpaperChanger</class>
 <widget class="QMainWindow" name="WallpaperChanger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>363</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WallpaperChanger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="browseBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="adressLned"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Location of wallpapers:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="changeBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Change wallpaper now</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="shuffleChk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Shuffle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>211</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Time between changes:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hourSpn">
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>5</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minSpn">
       <property name="maximum">
        <number>66</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="timeLeftLbl">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Awaiting for the program to load...</string>
    </property>
   </widget>
   <widget class="QLabel" name="wallCountLbl">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No wallpapers loaded...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionExit_to_tray"/>
    <addaction name="acionChange_on_open"/>
    <addaction name="actionChange_on_save"/>
   </widget>
   <addaction name="menuConfig"/>
  </widget>
  <action name="actionExit_to_tray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit to tray</string>
   </property>
  </action>
  <action name="acionChange_on_open">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change on opening</string>
   </property>
  </action>
  <action name="actionChange_on_save">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change on save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
